quickcheck-dynamic.cabal

```diff
@@ -1,63 +1,80 @@
-cabal-version: 2.2
-name: quickcheck-dynamic
-version: 0.1.0.0
-license: Apache-2.0
+cabal-version:   2.2
+name:            quickcheck-dynamic
+version:         0.1.0.0
+license:         Apache-2.0
 license-files:
   LICENSE
   NOTICE
-maintainer: ulf.norell@quviq.com
-author: Ulf Norell
-homepage: https://github.com/iohk/plutus#readme
-bug-reports: https://github.com/iohk/plutus/issues
+
+maintainer:      ulf.norell@quviq.com
```

Comment:
This should maybe be IOHK now.

Reply:
I have asked  Ulf and Maximilian to join the project in order to get their approval.